tensorflow/python/kernel_tests/linalg/sparse/csr_sparse_matrix_ops_test.py

```diff
@@ -533,8 +545,17 @@ def testSparseMatrixMatMulConjugateOutput(self):
 
   @test_util.run_in_graph_and_eager_modes
   def testLargeBatchSparseMatrixMatMul(self):
+    dtypes_to_test = [np.float32]
+    if not test.is_built_with_rocm():
+      # complex types is not supported on the ROCm platform
+      dtypes_to_test += [np.complex64]
+
+    if test.is_built_with_rocm():
+      # TODO(rocm): fix this
```

Comment:
Could you elaborate a little?

Reply:
will push out a commit soon with a couple of lines of added comments.
basically these subtests are failing on the ROCm platform, and need to be re-enabled once the underlying fix becomes available.